HandsOn/Group16/mappings/yarrrml_mapping.yml

```diff
@@ -0,0 +1,40 @@
+prefixes:
+  ns: "http://www.semanticweb.org/group16/ontologies/air-quality#"
+  rdfs: "http://www.w3.org/2000/01/rdf-schema#"
+  sc: "http://www.schema.org#"
+
+mappings:
+    
+    TriplesMap1:
+        sources:
+            - [csv-updated.csv~csv, "$1"]
+            - [estaciones.csv~csv, "$2"]
+        s:
+            - http://www.semanticweb.org/group16/ontologies/air-quality/$1(PUNTO_MUESTREO)
+        po:
+            - [a, ns:Station]
+            - [ns:stationCode, $1(PUNTO_MUESTREO)]
+            - [rdfs:label, $2(NOMBRE)]
+            - [sc:address, $2(DIRECCION)]
+            - [sc:latitude, $2(LATITUD)]
+            - [sc:longitude, $2(LONGITUD)]
+            - p: ns:Station
+              o:
+                - mapping: TriplesMap2
+                  condition:
+                    function: equal
+                    parameters:
+                      - [str1, $1(PUNTO_MUESTREO)]
+                      - [str2, $2(ESTACION_ID)]
+    
+    TriplesMap2:
+        sources:
+            - [csv-updated.csv~csv, "$1"]
+        s:
```

Comment:
s: http://www.semanticweb.org/group16/ontologies/air-quality/$1(ID) (in one line)